src/main/resources/application.yaml

```diff
@@ -75,7 +75,7 @@ featureFlag:
   isEmStitchingEnabled: ${IA_EM_STITCHING_ENABLED:false}
   isSubmitHearingRequirementsEnabled: ${IA_SUBMIT_HEARING_REQUIREMENTS_ENABLED:true}
   timedEventServiceEnabled: ${IA_TIMED_EVENT_SERVICE_ENABLED:false}
-  isfeePaymentEnabled: ${IA_FEEPAYMENT_ENABLED:false}
+  isfeePaymentEnabled: ${IA_FEEPAYMENT_ENABLED:true}
```

Comment:
we need to remember to change to `false` before merging to master

Reply:
I've reset the feature flag to false.